templates/santa/exclusions.html.twig

```diff
@@ -0,0 +1,218 @@
+{% extends 'santa/base.html.twig' %}
+
+{% set currentStep = 2 %}
+
+{% import "santa/application/exclusions_" ~ application ~ ".html.twig" as macros %}
+
+{% block santa_content %}
+    <div class="run-block">
+        {{ form_start(form, {
+            'attr': {'id': 'run_form'},
+            'method': 'POST',
+            'action': path('exclusions', { application: application })
+        }) }}
+            {# Prevent implicit submission of the form - no need for JS here (see https://stackoverflow.com/a/51507806/1917092) #}
+            <button type="submit" disabled style="display: none" aria-hidden="true"></button>
+
+            <h2>Define some exclusions</h2>
+
+            {% if not form.vars.valid %}
+                <span class="error">
+                    <i class="fas fa-exclamation-triangle"></i>
+                    {% for error in errors %}
+                        {% if not loop.first %}
+                        <br>
+                        {% endif %}
+                        {{ error }}
+                    {% endfor %}
+                </span>
+            {% endif %}
+
+            <div class="user-field message-step">
+                <p>
+                    Add any exclusions you want to apply to the Secret Santa.
+                    An exclusion indicates that a participant should not be assigned to offer
+                    a gift to another specific participant.
+                </p>
+
+                {% apply spaceless %}
+                    <div class="user-container">
+                        <div class="user-list no-hover">
+                            {% set key = 0 %}
+                            {% for child in form.exclusions %}
+                                {% set user = config.getUser(child.vars.name) %}
+                                <div class="user-item exclusions">
+                                    <div class="user-column">
+                                        <span class="user-data">
+                                            {{ macros.user_item(user, 32) }}
+                                        </span>
+                                        <span class="user-help">must not be assigned with</span>
```

Comment:
Not sure about the wording. But we have to make it extra clear about the direction of the exclusion.

```suggestion
                                        <span class="user-help">must not gift those persons</span>
```